### ISSUE TITLE
Error when I press 'Add to Slide'

### ISSUE THREAD

**Issue by isukrit** [brendena/MathEquationsGoogleSlide#15]
It gives me error: 'Error - ScriptError: Authorisation is required to perform that action.' I already provided the authorisation from my end while installing the add-on.

**Comment by mumbricht** [comment]
I have the same error.  I am using Linux and Firefox.  The error disappears when using windows 10 and microsoft edge.

**Comment by sometimescasey** [comment]
Just encountered today, Chrome + OSX. Any idea what's going on?

**Comment by mumbricht** [comment]
Nope, but it seems to have to do with operating system.  It worked fine
when I opened it up in windows.

**Comment by cromes** [comment]
Having same problem here. Linux with Chromium.

**Comment by talhajunaidd** [comment]
Same issue on linux chrome
